skyreader/plot/plot.py

```diff
@@ -825,8 +704,9 @@ def bounding_box(ra, dec, theta, phi):
             equatorial_map,
             coord='C',
             column_names=column_names,
+            column_units=["pix-1"],
```

Comment:
Maybe this unit should be set to a variable in the `if llh_map .. else` and then passed through here.